extensions/wikia/EmbeddableDiscussions/templates/DiscussionThreadDesktop.mustache

```diff
@@ -1,4 +1,4 @@
-<section class="module embeddable-discussions-module">
+<div class="module embeddable-discussions-module">
```

Comment:
Due to parser's logic, the section got wrapped in paragraphs `<p></p>`. Changing it to use div's solves the issue.